src/main/examples/maven_settings_ex/settings.xml

```diff
@@ -0,0 +1,20 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                              https://maven.apache.org/xsd/settings-1.0.0.xsd">
+
+    <profiles>
+        <profile>
+            <id>db-props</id>
+            <properties>
+                <db.username>postgres</db.username>
+                <db.password>postgres</db.password>

```

Comment:
Zaczynam sie zastanawiac :P
Moze jak czepiasz sie nazwy to podaj jakbys ja zmienil a nie tylko mowil ze jest zle i tyle XD